internal/app/summon/command/temp_factory.go

```diff
@@ -21,25 +19,43 @@ type TempFactory struct {
 // NewTempFactory creates a new temporary file factory.
 // defer Cleanup() if you want the files removed.
 func NewTempFactory(path string) TempFactory {
+	return NewCustomTempFactory(path, "")
```

Comment:
Can't remember if I mentioned this last time, but per the comment above, this should be `NewTempFileFactory` and the filename should be `temp_file_factory.go` and the change should be made whereever else it's relevant, since a `TempFactory` is too ambiguous.

Reply:
We discussed this to be pushed to another story: 
https://github.com/cyberark/secretless-broker/issues/721
https://conjurhq.slack.com/archives/CBTETAE20/p1559053808056500?thread_ts=1559052741.053300&cid=CBTETAE20